integration-tests/src/test/resources/model-in-image/miisection-toaddin-domain-template.yaml

```diff
@@ -1,5 +1,11 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# This file need to appended into kubernetes/samples/scripts/common/domain-template.yaml
+
+
   %MII_PREFIX%configuration:
-  # secrets: [ %DOMAIN_UID%-rcu-access ]
+    #%MII_PREFIX%  secrets: [ %WEBLOGIC_CREDENTIALS_SECRET_NAME% ]
```

Comment:
Why would we generate commented out content?

Reply:
That was a workaround for a WDT issue.  I removed the line on my latest commit as this is available in model-in-image